jest/helpers.ts

```diff
@@ -196,3 +196,10 @@ ${chalk.bold('stdout:')}  ${result.stdout}
 ${chalk.bold('code:')}    ${result.code}`);
   }
 }
+
+export function replaceProjectRootInOutput(output: string, testFolder: string) {
+  return output.replace(
+    new RegExp(`(:\\s")(.*)(\\/${testFolder}.*)`, 'g'),
```

Comment:
I think it would be more reliable to replace the `process.cwd()`? Also less regexy

Reply:
I mean, such helper may be helpful (e.g. when testing tmp directories) but in this case I think replacing process.cwd is all we need.

Reply:
Maybe the best way it's to connect both solutions and just pass `process.cwd()` as an second argument and then we would have `output.replace(testFolder, '<<REPLACED_ROOT>>')` because when we put just process.cwd it would be hard to test it ofc we can mock process.cwd but I think it's better WDYT?